### PR TITLE
[FEAT] - Add github workflows for pull request

### PR DESCRIPTION
**Pull request by rodrigooler**

### CODE REVIEW

**Comment by rodrigooler** [comment]
@steinerkelvin I'll leave it like this for now, but in the future I'll add cypress and typescript to check typing

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: yarn
+      - run: yarn format:check
```
Comment (steinerkelvin) [inline .github/workflows/format.yml:14]: image
ree

**Comment by steinerkelvin** [comment]
@rodrigooler It should be easy to add `tsc` checking.

File: .github/workflows/format.yml (hunk 1 above)
Comment (rodrigooler) [reply .github/workflows/format.yml:14]: ree?

**Comment by rodrigooler** [comment]
@steinerkelvin yes, but I preferred to close the PR right away to focus on the authentication part... then I'll adjust this and also configure the E2E better for some reason of the error, I'll see more calmly

**Pull request merged**